firefox_ui_tests/remote/security/test_unknown_issuer.py

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette import By
+from marionette.errors import MarionetteException
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestUnknownIssuer(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.test_data = 'https://ssl-unknownissuer.mozqa.com'
+
+    def tearDown(self):
+        try:
+            self.windows.close_all([self.browser])
+        finally:
+            FirefoxTestCase.tearDown(self)
+
+    def test_unknown_issuer(self):
+        self.marionette.set_context('content')
+
+        # Go to a site that has a cert with an unknown issuer
+        self.assertRaises(MarionetteException, self.marionette.navigate, self.test_data)
+
+        # Wait for about:error page
+        time.sleep(1)
+
+        # Check the issuer in cert_domain_link
+        issuer = (self.marionette.find_element(By.ID, 'cert_domain_link')).get_attribute(
+            'textContent')
+        self.assertEquals(issuer, 'ssl-selfsigned-unknownissuer.mozqa.com')
```

Comment:
Please separate the retrieval of elements and working with those. Here I would use the `get_attribute` call inside `assertEquals`.